app/views/devise/sessions/new.html.erb

```diff
@@ -24,3 +25,4 @@
 <% end %>
 
 <%= render "devise/shared/links" %>
+</div>
```

Comment:
Need a newline at the end of this file. Then you can merge.

Reply:
Taken care of thank you for the quick review!!